packages/fiori/src/ShellBar.ts

```diff
@@ -29,387 +49,49 @@ import {
 	SHELLBAR_OVERFLOW,
 } from "./generated/i18n/i18n-defaults.js";
 
-// Templates
-import ShellBarTemplate from "./generated/templates/ShellBarTemplate.lit.js";
-import ShellBarPopoverTemplate from "./generated/templates/ShellBarPopoverTemplate.lit.js";
+type AccessibilityRoles = {
+	logoRole?: string;
+};
 
-// Styles
-import styles from "./generated/themes/ShellBar.css.js";
-import ShellBarPopoverCss from "./generated/themes/ShellBarPopover.css.js";
+type AccessibilityTexts = {
+	logoTitle?: string;
+	profileButtonTitle?: string;
+};
 
-const HANDLE_RESIZE_DEBOUNCE_RATE = 200; // ms
+type AccessibilityAttributesProperties= {
```

Comment:
`AccessibilityAttributesValues` is more appropriate. Because attributes and properties are kind of the same thing (or related to the same thing) and becomes confusing